Add 2.15 docs for authenticated private registries (#1235)
rancher/rancher-product-docs: versions/v2.15/modules/en/pages/cluster-admin/helm-charts-in-rancher/helm-charts-in-rancher.adoc

Rancher and Partner charts may have extra configurations available through custom pages or questions.yaml files. However, all chart installations can modify the values.yaml and other basic settings. After you click *Install*, a Helm operation job is deployed, and the console for the job is displayed.

[NOTE]
====

> If the cluster is configured to use an authenticated registry, Rancher automatically injects the required pull secrets into the Helm values of charts from the built-in Rancher catalog. This does not apply to charts which defined custom pull secrets during installation, or charts from custom repositories. For more information, see {xref-authenticated-private-registries}[Authenticated Private Registries].

Reviewer: ```suggestion
If the cluster uses an authenticated registry, Rancher automatically injects the required pull secrets into the Helm values for charts from the built-in Rancher catalog. This behavior does not apply to charts with custom pull secrets specified during installation or charts from custom repositories. For more information, see {xref-authenticated-private-registries}[Authenticated Private Registries].
```
